app/Party.php

```diff
@@ -39,7 +39,7 @@ class Party extends Model implements Auditable
         'shareable_code',
         'online',
     ];
-    protected $hidden = ['created_at', 'updated_at', 'deleted_at', 'frequency', 'group', 'group', 'idevents', 'user_id', 'wordpress_post_id'];
+    protected $hidden = ['created_at', 'updated_at', 'deleted_at', 'frequency', 'group', 'group', 'user_id', 'wordpress_post_id'];
```

Comment:
What's the reason for removing this?  Is it because we need idevents in an array somewhere?

Just wondering if it might have an unintended consequence somewhere else.  I guess probably not, as it is adding rather than removing a field from what is included in arrays/json.

Reply:
Yes, exactly.